css/sign.css

```
@@ -0,0 +1,104 @@
+* {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+
+body{
+    background-color: #ffffff;
+    margin:0;
+}
+
+.sign-banner {
+    display: flex;
+    justify-content: center;
+    gap: 60px;
+    margin: 60px auto; 
+}
+
+.sign-banner-img {
+    width: 369px;
+    height: 132px;
+}
+
+.sign-form {
+    display: flex;
+    flex-direction: column;
+    width: 640px;
+    margin: 40px auto;
+    gap: 24px;
+}
+
+.sign-form label {
+    display: block;
+    color: #1F2937;
+    font-size: 18px;
+    font-weight: 700;
+    margin-bottom: 16px;
+}
+
+.sign-form input {
+    background-color: #F3F4F6;
+    color: #9CA3AF;
+    font-size: 16px;
+    width: 640px;
+    height: 56px;
+    padding: 16px;
+    padding-left: 24px;
```

```suggestion
    padding: 16px 16px 16px 24px;
```

조금 귀찮지만 요렇게 써주도록 할까요...?! ㅋㅋ